app/src/main/java/com/jalasoft/routesapp/util/algorithm/RouteCalculator.kt

```diff
@@ -0,0 +1,51 @@
+package com.jalasoft.routesapp.util.algorithm
+
+import android.location.Location
+import com.jalasoft.routesapp.data.model.remote.AvailableTransport
+import com.jalasoft.routesapp.data.model.remote.LinePath
+import com.jalasoft.routesapp.data.model.remote.LinesCandidate
+
+object RouteCalculator {
+    fun calculate(lines: List<LinePath>, destination: Location, origin: Location, minDistance: Double, minDistanceBtwStops: Double): MutableList<AvailableTransport> {
```

Comment:
good job refactoring but still cognitive complexity for this method is a little bit big

Reply:
I've refactored, according to sonar lint file now doesn't contain any cognitive complexity